erizo_controller/erizoClient/src/Connection.js

```diff
@@ -49,18 +49,22 @@ Erizo.getBrowser = function () {
     } else if (window.navigator.userAgent.match('Bowser') !== null){
         browser = 'bowser';
     } else if (window.navigator.userAgent.match('Chrome') !== null) {
-        if (window.navigator.appVersion.match(/Chrome\/([\w\W]*?)\./)[1] >= 26) {
-            browser = 'chrome-stable';
+        browser = 'chrome-stable';
+        if (window.navigator.userAgent.match('Electron') !== null) {
```

Comment:
Yep, it contains both Electron and Chrome.
Actually the original idea was to add it as another `else if` but that does not work because of that.